browser/ui/webui/speedreader/speedreader_toolbar_ui.cc

```diff
@@ -41,8 +46,12 @@ SpeedreaderToolbarUI::SpeedreaderToolbarUI(content::WebUI* web_ui,
     source->AddString(str.name, l10n_str);
   }
 
+#if BUILDFLAG(ENABLE_AI_CHAT)
   source->AddBoolean("aiChatFeatureEnabled",
                      ai_chat::features::IsAIChatEnabled());
+#else
+  source->AddBoolean("aiChatFeatureEnabled", false);
```

Comment:
Speedreader has an "open ai chat" button

Reply:
I think it's ok to leave the frontend code in and only show it if the feature is enabled., especially since it's a very simple button. There's plenty of chromium precedent for that.